integration-tests/core/src/test/java/org/apache/camel/quarkus/core/CoreTest.java

```diff
@@ -33,12 +33,18 @@
 
 @QuarkusTest
 public class CoreTest {
+
     @Test
     public void testContainerLookupFromRegistry() {
         RestAssured.when().get("/test/registry/lookup-registry").then().body(is("true"));
         RestAssured.when().get("/test/registry/lookup-context").then().body(is("true"));
     }
 
+    @Test
+    public void testLookupRoutes() {
+        RestAssured.when().get("/test/routes/lookup-routes").then().body(is("true"));
```

Comment:
Comparing the list of routeIds here would make it easier to troubleshoot if something goes wrong.
```suggestion
        RestAssured.when().get("/test/routes/lookup-routes").then().body(is("foo,bar"));
```

Reply:
Yeah good idea